assets/js/script.js

```diff
@@ -119,9 +119,10 @@ function checkPlayerCards(hand) {
 
 // Event listener to generate decks
 var generateDeckEl = document.querySelector("#generateDeck");
+var removeButton = document.querySelector("#remove-button");
```

Comment:
NIT: to keep consistent with out naming convention, please use camel case: "removeButton"